src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -303,6 +303,7 @@ define([
         },
         previewStep:function () {
             // no need for validation if going to composer
+            this.currentView.model.spec.set("config", this.currentView.getConfigMap());
```

Comment:
Could you just add a comment stating what this does please? Will be useful for the future